README.md

```diff
@@ -98,6 +98,10 @@ The following have been tested with Debian 8, it should work on Ubuntu as well.
 * `gitea_db_ssl`: Use SSL ? (postgres only!). Can be `required`, `disable`, `verify-full`
 * `gitea_db_path`: DB path, if you use `sqlite3`. The default is good enough to work though.
 
+### Fail2Ban configuration
+
+* `gitea_install_fail2ban_config`: Wether to deploy the fail2ban config snippets
```

Comment:
Maybe specify explicitly in the doc that fail2ban has to be preexisting on the machine, otherwise the module will fail